routes/auth.js

```diff
@@ -2,11 +2,11 @@ var authController = require('../controllers/authcontroller.js');
 
 module.exports = function(app,passport){
 
-app.get('/landing', authcontroller.landing);
+app.get('/landing', authController.landing);
```

Comment:
This was defined with authController above so we needed to use the same reference here.